kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -321,9 +307,17 @@ def get_path(self) -> Path:
         return self._env_path
 
     def install(self, settings: PythonSettings) -> None:
-        command = ["uv", "sync", "--all-extras", "--frozen"]
-        logger.info("%s", command)
-        sp.check_call(command, cwd=self.project_directory)
+        _run_uv_command_with_indexes(["uv", "sync", "--all-extras"], settings, self.project_directory)
 
     def always_install(self) -> bool:
         return True
+
+
+def _run_uv_command_with_indexes(command: list[str], settings: PythonSettings, work_dir: Path) -> None:
```

Comment:
This doesn't require the command to be a uv subcommand: can we rename it to `_run_with_uv_indexes` or similar?

Reply:
Good name suggestion! Implemented